util/reggen/validate.py

```diff
@@ -246,7 +245,7 @@ def bitmask(bfield):
         lsb = brange[2]
         res = 0
         if ((not msb.isdecimal()) or (not lsb.isdecimal()) or
-            (int(lsb) > int(msb))):
+                (int(lsb) > int(msb))):  # yapf: disable
```

Comment:
This indentation looks strange to me. I'd expect it to line up with the `(not ...` on the line above. Is this just Yapf weirdness?

Reply:
If lined up, flake9 raises an error, if indented, no flake8 but yapf fixed it to line up.